src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -556,7 +556,7 @@ public DisplayVersion SelectedVersion
                             PackageMetadata = detailedPackageMetadata;
                         }
 
-                        NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(() => SelectedVersionChangedAsync(_searchResultPackage, _selectedVersion.Version, loadCts.Token).AsTask());
+                        NuGetUIThreadHelper.JoinableTaskFactory.Run(() => SelectedVersionChangedAsync(_searchResultPackage, _selectedVersion.Version, loadCts.Token).AsTask());
```

Comment:
This could have perf side-effects. I'd suppress it.

Reply:
I'd use `.PostOnFault(..)` on all instances, rather than suppressing it. As I mentioned in the data meeting on Monday, we're not capturing real customer impact issues because of these not-awaited async methods.

Reply:
I switched to PostOnFailure for all of these calls. But the PostOnFailure method is one of the offending methods. That one I've suppressed. Is there something better we can do there? If not, I'll close the issue I created -- https://github.com/NuGet/Client.Engineering/issues/956